tests/test_client.py

```diff
@@ -1,46 +1,51 @@
+import pytest
 import concurrent
-from unittest.mock import ANY, MagicMock
-
-from google.cloud.pubsub_v1 import PublisherClient
-
-from rele import Publisher
-
-from . import settings
+from unittest.mock import ANY, patch
 
 
+@pytest.mark.usefixtures('publisher')
 class TestPublisher:
-
-    def test_returns_future_when_published_called(self):
+    @patch('time.time', return_value=1560244246.863829)
```

Comment:
We use to do this kind of patching (datetime, time) using a pytest fixture.

For instance:
```
@pytest.fixture
def published_at():
  return 1560244246.863829
@pytest.fixture
def time_mock(published_at):
  with patch('time.time') as mock:
    mock.return_value = published_at
    yield mock
```
Later on, as mock is not going to be referred to, but `published_at` will do:
```
@pytest.mark.usefixtures('publisher, time_mock')
def test(self, published_at, publisher):
...
assert_called_with(... published_at=published_at ...)
```
Furthermore, both fixtures will be very useful and reusable.

Reply:
Cool, I'm gonna do that. Thanks!